src/main/scala/singleton/ops/package.scala

```diff
@@ -127,8 +127,18 @@ package object ops {
   type &&[P1, P2]           = OpMacro[OpId.&&, P1, P2, NP]
   type ||[P1, P2]           = OpMacro[OpId.||, P1, P2, NP]
   type Substring[P1, P2]    = OpMacro[OpId.Substring, P1, P2, NP]
+  type SubSequence[P1, P2, P3] = OpMacro[OpId.SubSequence, P1, P2, P3]
+  type StartsWith[P1, P2]   = OpMacro[OpId.StartsWith, P1, P2, NP]
```

Comment:
In cases where the arguments are not trivial, lets give them a proper name. E.g.
`StartsWith[S, Prefix]`

Reply:
good point, I renamed them to be a bit more informative